data-prepper-api/src/test/java/org/opensearch/dataprepper/expression/ExpressionEvaluatorTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.hamcrest.CoreMatchers.equalTo;
+
+public class ExpressionEvaluatorTest {
+    private ExpressionEvaluator expressionEvaluator;
+    class TestExpressionEvaluator implements ExpressionEvaluator {
+        public Object evaluate(final String statement, final Event event) {
+            return event.get(statement, Object.class);
+        }
+    }
+
+    @Test
+    public void testDefaultEvaluateConditional() {
+        expressionEvaluator = new TestExpressionEvaluator();
+        assertThat(expressionEvaluator.evaluateConditional("/status", event("{\"status\":true}")), equalTo(true));
+        
+    }
+
+    @Test
+    public void testDefaultEvaluateConditionalThrows() {
+        expressionEvaluator = new TestExpressionEvaluator();
+        assertThrows(ClassCastException.class, () -> expressionEvaluator.evaluateConditional("/status", event("{\"status\":200}")));
```

Comment:
This raises an interesting point which I didn't consider.

By combining the expression evaluator, there really is no way to evaluate one expression with some conditional goal in mind versus evaluating with an object in mind.

I don't think that should hinder this PR because I think it could be resolved with some internal refactoring.

Reply:
We already have a way to check for existence `/status == null` would check for key exists or not. As David said, that functionality is not new.

Reply:
Yes that does work, I'm just saying I think `/status` as an alternative to `/status != null` is cleaner.